oracle/config/README.md

```diff
@@ -1,26 +1,26 @@
 # Configurations
 
-> To read more about how to create/configure a new provider to your oracle, please read the [Base Provider](../../providers/base/README.md) readme. This readme overviews how to configure the oracle side-car as well as how to hook it up to your application.
+> This readme overviews how to configure the oracle side-car as well as how to hook it up to your application. To see an example of a properly configured oracle-side car, please visit the [local config](./../../config/local) files - `oracle.json` and `market.json`. To see an example of a properly configured application, please visit the test application's [app.toml](./../../tests/simapp/slinkyd/testappd/root.go) generation code. Otherwise, please read on to learn how to configure the oracle side-car and application.
```

Comment:
```suggestion
> This readme overviews how to configure the oracle side-car as well as how to hook it up to your application. To see an example of a properly configured oracle-side car, visit the [local config](./../../config/local) files - `oracle.json` and `market.json`. To see an example of a properly configured application, please visit the test application's [app.toml](./../../tests/simapp/slinkyd/testappd/root.go) generation code. Otherwise, read on to learn how to configure the oracle side-car and application.
```